fern/snippets/version-number-swift.mdx

```diff
@@ -0,0 +1 @@
+0.7.0
```

Comment:
This is intentionally 1 version behind current so that I can validate that the CI for updating versions works once this is merged to main.